src/apiCalls.js

```diff
@@ -29,7 +29,6 @@ async function getDepartmentObjects(objectIDs, maxIDs = 40, setDepartmentObj) {
       const response = await fetch(`https://collectionapi.metmuseum.org/public/collection/v1/objects/${objectId}`);
```

Comment:
Have you considered pulling this fetch functionality declaration into an apiCalls file?  I know you mentioned the fetches feeling a little complicated so I can understand if you've deprioritized that.